web-client/src/views/TrialSessionWorkingCopy/PrintableTrialSessionWorkingCopyPreviewPage.jsx

```diff
@@ -0,0 +1,45 @@
+import { BigHeader } from '../BigHeader';
+import { Button } from '../../ustc-ui/Button/Button';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { PdfPreview } from '../../ustc-ui/PdfPreview/PdfPreview';
+import { WarningNotification } from '../WarningNotification';
+import { connect } from '@cerebral/react';
+import { sequences, state } from 'cerebral';
+import React from 'react';
+
+export const PrintableTrialSessionWorkingCopyPreviewPage = connect(
+  {
+    gotoTrialSessionWorkingCopySequence:
+      sequences.gotoTrialSessionWorkingCopySequence,
+    headerTitle: state.screenMetadata.headerTitle,
+    navigateBackSequence: sequences.navigateBackSequence,
```

Comment:
🔧 Looks like this isn't used, can it be removed?

Reply:
Resolved!